src/test/functional/features/admin-create-user.feature

```diff
@@ -11,28 +11,69 @@ Feature: create admin user
 
   Scenario: creating new user with valid e-mail and password
     When I enter Email "test@gmail.com"
-    When I enter First Name "testuser"
-    When I enter Last Name "lastname"
-    Then I select the user role as fact-admin
+    When I enter First Name "testuser" to create new user
+    When I enter Last Name "lastname" to create new user
+    Then I select the user role as fact-admin for new user
     Then I click create user button
     When I enter Password "Pa55word11"
     Then I click confirm button
     Then I can see green success message "User invited successfully"
 
   Scenario: creating new user with duplicate e-mail
     When I enter Email "hmcts.super.fact@gmail.com"
-    When I enter First Name "test"
-    When I enter Last Name "test"
-    Then I select the user role as fact-admin
+    When I enter First Name "testuser" to create new user
+    When I enter Last Name "lastname" to create new user
+    Then I select the user role as fact-admin for new user
     Then I click create user button
     When I enter Password "Pa55word11"
     Then I click confirm button
     Then The error message display for creating user "User with this email already exists."
 
   Scenario: creating new user with blank entries
     When I enter Email "test@gmail.com"
-    When I enter First Name ""
-    When I enter Last Name ""
-    Then I select the user role as fact-admin
+    When I enter First Name "" to create new user
+    When I enter Last Name "" to create new user
+    Then I select the user role as fact-admin for new user
     Then I click create user button
     Then The error message display for creating user "All fields are required."
+
+  Scenario: Updating test user role
+    When I click on edit user
+    When I enter User Email "test@fact.com"
```

Comment:
You should be able to update the https://tools.hmcts.net/confluence/display/DATS/FaCT+Admin+Portal page. Can we add a mini section in there to mention that we have added a test@fact.com user for these tests? Just incase people wonder what the use case was for this on the IDAM side and query future peeps on our project